packages/editor/src/extensions/outline-list/outline-list.ts

```diff
@@ -98,9 +98,10 @@ export const OutlineList = Node.create<OutlineListOptions>({
   },
 
   addNodeView() {
-    return () => {
+    return ({ HTMLAttributes }) => {
       const ul = document.createElement("ul");
       ul.classList.add("outline-list");
+      if (HTMLAttributes.dir) ul.dir = HTMLAttributes.dir;
```

Comment:
`HTMLAttributes` might not always contain `dir` so you should add a fallback here like `node.attrs.textDirection`.